### DISCUSSION TITLE
Unable to upload i20

### DISCUSSION THREAD

**Discussion by gnpthbalaji** [community/community#64143] [GitHub Education]
### Hi everyone,

I am unable to upload my I-20 since my university ID doesn't have the Program End Date. When I try to take a picture of the I20, It doesn't read the University Name. 

How do I upload the document instead of taking a picture? I tried with the Phone, but it doesn't switch the camera to the back camera. 

Please Let me know.

<img width="733" alt="image" src="https://github.com/community/community/assets/59932973/cd34265a-2e67-4403-964b-ea3563fc1647">

**Comment by GhostOf0days** [comment]
Take a screenshot and upload that.

**Reply by gnpthbalaji** [reply]
There is no upload button only "Take a Photo"
